frontend/src/components/pages/pastShifts.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Container } from '@mui/material';
+import { AiOutlineMenu } from 'react-icons/ai';
```

Comment:
Also, I forgot to mention, feel free to remove the menu icon. The task for this week will add it to the necessary pages 👍